FIRO_TSEnsembles/src/main/java/hec/stats/MedianComputable.java

```diff
@@ -0,0 +1,16 @@
+package hec.stats;
+
+import java.util.Arrays;
+
+public class MedianComputable implements Computable {
+    @Override
+    public float compute(float[] values) {
+        //calculate the min of values
+        int n = values.length;
+        Arrays.sort(values);
+        if(n % 2 !=0) {
+            return values[n /2];
+        }
+        return (values[(n-1)/2] + values[n /2]) / 2;
```

Comment:
Do you have tests for this yet? I am suspicious that the rounding down is the default, not rounding up. i think you are hoping for rounding up?

Reply:
you may have to cast the result of n/2 to int for this to work.

Reply:
tests added.  I believe its working as intended